_docs/reference/release-roadmap.md

```diff
@@ -12,7 +12,7 @@ type: markdown
 
 Istio 0.2 will be our second release, and will work across multiple Kubernetes clusters and some basic hybrid deployments. Istio 0.2 is focused on expanding the coverage of Istio to more places, and making it easier for users and vendors to integrate into the system. Istio 0.2 will include the following features:
 - TCP telemetry collection and policy enforcement.
-- Hybrid deployments (multiple clusters, VMs).
+- Initial Raw VM support.
```

Comment:
Sure, but we are doing VM only mode, and k8s+VMs. That is still hybrid deployment.